File: Client/Captions/CaptionsLib/source/tts/TTSReader.vb
'Filename: TTSReader.vb
'Version: 20121015

Imports CaptionsLib
Imports CaptionsLib.Models
Imports CaptionsLib.TTS.TTSUtils

Imports System.IO
Imports System.Text

Namespace CaptionsLib.TTS

  Public Class TTSReader
    Inherits BaseCaptionReader

    Public Overrides Sub ReadCaption(ByVal caption As ICaption, ByVal reader As TextReader)
      TTSStringToCaption(reader.ReadLine(), caption)
    End Sub

  End Class

End Namespace

File: Client/Captions/CaptionsLib/source/tts/TTSReader.cs
//Filename: TTSReader.cs
//Version: 20131105

using Microsoft.SilverlightMediaFramework.Core.Accessibility.Captions;
using System.IO;

namespace ClipFlair.CaptionsLib.TTS
{

	public class TTSReader : BaseCaptionReader
	{

		public override void ReadCaption(CaptionElement caption, TextReader reader)
		{
			TTSUtils.TTSStringToCaption(reader.ReadLine(), caption);
		}

	}

}